<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Satellite TLE Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="urlLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите URL для загрузки данных TLE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadUrlButton">
      <property name="text">
       <string>Загрузить по URL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectFileButton">
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="statsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveStatsButton">
      <property name="text">
       <string>Сохранить статистику</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Статус: Готово</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
